### ISSUE TITLE
Week 4 annoucements

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#29]
## The TLDR this week:

- [Week 4 todo](https://github.com/campus-experts/spring-2017/blob/master/todos/week-4.md)
- [Last week's video](https://www.dropbox.com/s/086yd5pq0m3e82f/gittalkcut.mp4?dl=0), and last week's [tech talk](https://www.dropbox.com/s/nb6n9f9kr8nre1a/graphql.mp4?dl=0).
- This week's exercise is - [writing READMEs](https://github.com/campus-experts/spring-2017/blob/master/docs/write-a-readme.md)! Open an issue with your README review or link to your new README.
- Check out and comment on other people's [impact propsals](https://github.com/campus-experts/spring-2017/pulls?q=is%3Apr+is%3Aopen+label%3Aimpact-proposal).
- This week we have two guest talks! Technical Writing on 18th April, and Music Computing tech talk on 20th April. 

## Welcome to Week 4 🚩

You've made it to week 4! Phew! We're past the Community Assessment and Impact Proposals now, which are the most important and biggest exercises in the program. If you haven't submitted those yet, please do so as soon as possible. From here on out, the exercises are a bit lighter, primarily to reinforce the content you're learning in the training talks. 

This week we've acted on your feedback about discussions, and you can now participate over in the [discuss repo](https://github.com/campus-experts/discuss). We'll be setting a topic each week, but feel free to open other discussions relevant to you or comment on discussions from previous weeks. 

[This week's tech talk](https://github.com/campus-experts/spring-2017/issues/28) is Hugh Rawlinson! Hugh works on developer relations over at Spotify, and whilst he's in the UK for a hackathon, we took the opportunity to grab him. Hugh is a music computing and web audio API wizz and he'll be giving us a run down of all the ways you can make computers bloop on Thursday. 

## This week on Slack
Missed a discussion on slack? Not to worry!

- Done a [live coding challenge event](https://githubeducation.slack.com/archives/C2TQDRQSG/p1492379799023727)?
- Pandelis is looking for feedback on his code of conduct,
 and has got some [great sources to draw from](https://github.com/campus-experts/spring-2017/issues/23)
- Elliot made an awesome [Slack theme!](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491923969145205)
